### ISSUE TITLE
How to deal with "Cannot find word list en.utf-8.spl or en.ascii.spl" warning

### ISSUE THREAD

**Issue by unphased** [neovim/neovim#2102]
I found that on my Linux system, 

```
$ sudo find / -name en.utf-8.spl        
/usr/share/vim/vim74/spell/en.utf-8.spl  
```

This is obviously not a place that neovim should or would be looking at.

What is a good way to go about fixing this issue? Did i bork my install? I just ran `make; make install` and it went swimmingly.

I am using .nvimrc which sources my .vimrc. I don't know if it is possible in the nvimrc to tell it to look for en.utf-8.spl in `/usr/share/vim/vim74/spell`.

I did not create a symlink for the directory .nvim -> .vim. When I tried this, it did not fix anything.

**Comment by unphased** [comment]
Actually I think I just solved this by just using neovim, it quickly prompted me to download the files and even asked me where to place it. So NeoVim handled this very naturally. I don't know what about the C++ file I opened that triggers this (which my .vimrc and .nvimrc files did not trigger the spell file download).

**Comment by unphased** [comment]
BTW, as a heavy Vim user, I am simply *astounded* to find out that neovim replicates Vim's functionality so exactly that I have had to make zero changes to my near-2000 line vimrc and ALL plugins including YouCompleteMe appear to be functioning flawlessly. I don't know how you guys did it. This shouldn't have been possible. Mad props.

**Comment by cgbeutler** [comment]
I just ran into this issue. If you set the language without turning on spell, you will get the error without getting the auto-download stuff. I have these lines in my .vimrc (copied to .nvimrc) that caused the problem:

    if version >= 700
        set spl=en spell
        set nospell
    endif

calling `:set spell` by hand from within nvim then triggered the downloading of the word lists.

**Comment by nathanaelkane** [comment]
No matter what I do, `:set spell` never prompts me to download the file.

**Comment by tex** [comment]
I found that some plugins for enhancing spell capability somehow disable the prompt. I started `nvim -u none` then run `:set spell` and `nvim` prompted me to download the file.

**Comment by fennecdjay** [comment]
I hit similar issue with neovim from git on archlinux.
However, copying vim's spell directory in ~/.config/nvim/spell did the trick.

**Comment by justinmk** [comment]
With `nvim -u NONE` the prompt won't appear because plugins are disabled then. It should appear with `nvim -u NORC`.

**Comment by tex** [comment]
Hello,

any idea what to do with this? The folder is there and is writeable.

![obrazek](https://cloud.githubusercontent.com/assets/27386/26189240/cf35d6b6-3ba3-11e7-933b-a9b3743497fd.png)

NVIM v0.2.1-6-g9b1cad7c
Build type: Release
Compilation: C:/msys64/mingw64/bin/gcc.exe -Wconversion -DNVIM_MSGPACK_HAS_FLOAT
32 -O2 -DNDEBUG -DDISABLE_LOG -Wall -Wextra -pedantic -Wno-unused-parameter -Wst
rict-prototypes -std=gnu99 -D__USE_MINGW_ANSI_STDIO -D_WIN32_WINNT=0x0600 -Wvla
-fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -IC:/projects/neovim/b
uild/config -IC:/projects/neovim/src -IC:/projects/neovim/.deps/usr/include -IC:
/projects/neovim/.deps/usr/include -IC:/projects/neovim/.deps/usr/include -IC:/m
sys64/mingw64/include -IC:/msys64/mingw64/include -IC:/projects/neovim/build/src
/nvim/auto -IC:/projects/neovim/build/include
Compiled by appveyor@APPVYR-WIN

Optional features included (+) or not (-): -acl   +iconv    -jemalloc -tui

For differences from Vim, see :help vim-differences

   system vimrc file: "$VIM\sysinit.vim"
  fall-back for $VIM: "C:/Program Files (x86)/nvim/share/nvim"

**Comment by lewis6991** [comment]
I've also started seeing this issue again. Using version `NVIM v0.2.1-780-g5b32bce73`

**Comment by shinzui** [comment]
Is there a way to automate the download from a script without user interaction?

**Comment by maker43** [comment]
> 

Hello I had a same problem when running nvim. My soulution was that you run nvim as sudo user.
First in the terminal you type: `sudo nvim`
Then in nvim you write `:set spelllang=en spell`
after that nvim did download spellfile properly after asking me do I agree.
Hope this helps.

**Comment by aloispichler** [comment]
Thank you indeed. This solved the problem here as well!

**Comment by melMass** [comment]
The spell index is also "open" in case you need to do it manually:
https://ftp.nluug.nl/pub/vim/runtime/spell

**Comment by oldbuilding** [comment]
I tried many things but so far have not found a way to use an existing dictionary.   For example, I tried using the en.utf-8.spl binary file that I found inside brew's installation files for neovim.  

```
mkdir ~/.dotfiles.git/.config/${NVIM_APPNAME:-astronvim}/spell
cp /opt/homebrew/Cellar/neovim/0.10.0/share/nvim/runtime/spell/en.utf-8.spl ./spell/
```

I can then attempt to associate that spl file with spell settings but no matter what, neovim ignores it entirely.  Calling 'spellgood word' for the first time within nvim results in a new binary spell file being created.

So this is the best I get after hours of experimenting.  

```
mkdir ~/.dotfiles.git/.config/${NVIM_APPNAME:-astronvim}/spell

nvim --headless -c "set spellfile=~/.config/${NVIM_APPNAME:-astronvim}/spell/en.utf-8.add" -c "mkspell! ~/.config/${NVIM_APPNAME:-astronvim}/spell/en.utf-8.spl" -c "qall"

nvim --headless -c "spellgood noice" -c "qall"
```

This is the only way that I've found that actually adds a custom word into my add.spl file.  Every other approach resulted in 'could not find file' errors.

`mkspell` creates the binary file.  I haven't found a way to force neovim to defer to an existing spl file.  Not calling `mkspell` during setup doesn't help; the new spl file will still be created when first adding words to the spell list.

* fwiw, I'm using AstroNvim with lazy.nvim.
